mothership-ec/cog-mothership-commerce
# Feature/delete addresses

#### What does this do?

Adds 2 new columns to order_address table and populates any missing data,
Marks addresses as deleted when a new one is created.
Doesn't load deleted addresses by default.
#### How should this be manually tested?

Make an order to check saving a new address works correctly.
Change the address a couple of times in fulfilment (post stage) and check deleted_at & deleted_by are populated.
Check this new address is used.
#### Related PRs / Issues / Resources?

Closes https://github.com/messagedigital/cog-mothership-commerce/issues/361
Reporting relies on this. 
#### Anything else to add? (Screenshots, background context, etc)
